docs/hypernode-deploy/applications/config-for-magento-2.md

```diff
@@ -17,6 +17,19 @@ return $configuration;
 
 By using the Magento2 ApplicationTemplate, a bunch of default configuration gets set in Hypernode Deploy, and should work out-of-the-box for most magento 2 stores.
 
+## Known issues
```

Comment:
I'd rephrase it to Common issues, because these issues are not related to Hypernode Deploy.